SDK_dotnet/Clients/ExchangeClient.cs

```diff
@@ -1094,8 +1094,8 @@ string privateKey
 
             if (currency.IsGluwacoinSideChainCurrency())
             {
-                convertAmount = GluwacoinConverter.ConvertToGluwacoinSideChainBigInteger(amount);
-                convertFee = GluwacoinConverter.ConvertToGluwacoinSideChainBigInteger(fee);
+                convertAmount = GluwacoinConverter.ConvertToGluwacoinSideChainBigInteger(amount, 6);
```

Comment:
sUSDCG and sNGNG have different decimals

sUSDCG is 6 decimals
sNGNG is 18 decimals

Reply:
This is handled in SDK_dotnet/Clients/GluwaClient.cs

Reply:
sNgNg only supports transaction. So the ExchangeClient doesn't need this. Also there is already validation for sNgNg currency in Exchange. However, it can be confusing when looking at the function alone, so I will make the number of digits const.

Reply:
> sNgNg only supports transaction. So the ExchangeClient doesn't need this.

Will confirm this with @taelimoh and Kam, as my understanding is that we will be adding sNGNG exchange support.

> Also there is already validation for sNgNg currency in Exchange

Can you tell me where? I couldn't find it

Reply:
https://gluwa.slack.com/archives/CKR3180JY/p1617344161188400
The gluwa api does not fully support sNgNg exchange yet. When done, I'll add it to this SDK.

I would like to leave the validation part as a comment. But I'm not familiar with GitHub. IsGluwaExchangeCurrency() <- This method validates.